lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -130,6 +130,12 @@ <h3 class="hd hd-3">${_("Regenerate Certificates")}</h3>
                 </div>
                 <div>
                     <label>
+                        <input id="certificate_status_verified_users_with_audit_certs" type="checkbox" name="verified_with_audit_certs" value="verified_users_with_audit_certs">
```

Comment:
Because [`CertificateStatuses`](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/certificates/models.py#L76-L99) does not contain `'verified_users_with_audit_certs'` – [link](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L339).